sycl/test-e2e/Matrix/SG32/joint_matrix_bfloat16_packedB.cpp

```diff
@@ -1,4 +1,4 @@
-//==----- joint_matrix_bfloat16_packedB.cpp  - DPC++ joint_matrix----------==//
+//==----- SG32/joint_matrix_bfloat16_packedB.cpp  - DPC++ joint_matrix----------==//
```

Comment:
It doesn't seem that `ACC_BFLOAT16` is being run for packed B case under SG32. Should that case be added?